suite/spdz/src/main/java/dk/alexandra/fresco/suite/spdz/datatypes/SpdzTriple.java

```diff
@@ -49,52 +49,4 @@ public SpdzSInt getC() {
   public String toString() {
     return "SpdzTriple [elementA=" + elementA + ", elementB=" + elementB + ", elementC=" + elementC + "]";
   }
-
```

Comment:
Not sure, do we not need equals and hash code?

Reply:
No?